kong/plugins/opentelemetry/otlp.lua

```diff
@@ -72,9 +77,39 @@ local function transform_events(events)
   return pb_events
 end
 
+-- this function is to translate the span to otlp span, but of universal span format
+local function translate_span(span)
+  local trace_id = span.trace_id
+  local len = #trace_id
+  local new_id = trace_id
+
+  if len == TRACE_ID_LEN then
+    return span
+  end
+
+  -- make sure the trace id is of 16 bytes
+  if len > TRACE_ID_LEN then
+    new_id = trace_id:sub(-TRACE_ID_LEN)
+
+  elseif len < TRACE_ID_LEN then
+    new_id = NULL:rep(TRACE_ID_LEN - len) .. trace_id
+  end
+
+  local translated = shallow_copy(span)
+  setmetatable(translated, getmetatable(span))
+
+  translated.trace_id = new_id
+  span = translated
+
+  return span
```

Comment:
Nitpick, not needed if we don't do a shallow copy.
```suggestion
  return translated
```